src/main/scala/partitioners/PriorityPartitioner.scala

```diff
@@ -0,0 +1,32 @@
+package partitioners
+
+import org.apache.kafka.clients.producer.Partitioner
+import org.apache.kafka.common.Cluster
+import org.apache.kafka.common.utils.Utils
+
+class PriorityPartitioner extends Partitioner {
+  private val partitionsPerPriority = 2
```

Comment:
@omerlh can you help me with syntax here? I don't get how to read from configuration, and I tried..